operations/reservations.md

```diff
@@ -28,11 +31,12 @@ Returns all reservations from the specified interval according to the time filte
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `TimeFilter` | string [Reservation time filter](reservations.md#reservation-time-filter) | optional | Time filter of the interval. If not specified, reservations `Colliding` with the interval are returned. |
+| `GroupIds` | array of string | optional | Unique identifiers of [Reservation group](#reservation-group). |
+| `TimeFilter` | string [Reservation time filter](#reservation-time-filter) | optional | Time filter of the interval. If not specified, reservations `Colliding` with the interval are returned. |
```

Comment:
This is the deprecated format, I will open new PR to fix it, the same for rows around.